### ISSUE TITLE
How to use LSTM with categorical output?

### ISSUE THREAD

**Issue by vinhqdang** [fchollet/keras#1248]
In the example at:

https://github.com/fchollet/keras/blob/master/examples/imdb_bidirectional_lstm.py

The output is integer and after predicting, we need to round the output to have a prediction.

Is there a way to use this example with categorical output, instead of converting categorical to integer and then convert back?

**Comment by olegsinyavskiy** [comment]
Here is the example of using single GRU to classify using categorial labels:
https://github.com/fchollet/keras/blob/master/tests/test_tasks.py#L55

Also, I think you can change activation of Dense to softmax and use it with categorical_crossentropy for your case

**Comment by vinhqdang** [comment]
Yeah, I applied the example of reuters with MLP and it works. Thanks a lot,
